apps/vs-code-designer/src/app/utils/bundleFeed.ts

```diff
@@ -104,15 +98,10 @@ export async function getDependenciesVersion(context: IActionContext): Promise<I
  * @param {IHostJsonV2} hostJson - Host.json configuration.
  */
 export async function addDefaultBundle(context: IActionContext, hostJson: IHostJsonV2): Promise<void> {
-  let versionRange: string;
```

Comment:
Do we really need to remove the `getLatestVersionRange`?

Reply:
No, we do not need to remove that. Mistake on my part.